src/core/utils/useReservations.tsx

```diff
@@ -0,0 +1,100 @@
+import { useGetReservationsV2 } from "../fbs/fbs";
+import { useGetV1UserReservations } from "../publizon/publizon";
+import { getQueuedReservations } from "./helpers/general";
```

Comment:
Please move this function from the general helpers to this file.

I do not see a reason why developers should use it directly.